HW0/ans/P6.txt

```diff
@@ -0,0 +1,7 @@
+A. (1) By coding it for several times, the results are different each time. The results printed can be grouped every four items. In each group, the set of the four items are the same each time while the order of the four items are random.
```

Comment:
I also agree. There can be times where the later task can be executed before the earlier one.